query/aql_processor_test.go

```diff
@@ -2342,4 +2370,18 @@ var _ = ginkgo.Describe("aql_processor", func() {
 		qc.initializeNonAggResponse()
 		Ω(w.Body.String()).Should(Equal(``))
 	})
+
+	ginkgo.It("Copy array to device should work", func() {
```

Comment:
this test doens't really copy anything to device, right..?
seems it's actually testing `readDeviceVPSlice`  which is a test util function